src/rest/connector/utils.py

```diff
@@ -40,4 +46,90 @@ def get_token(connection):
         except Exception:
             pass
 
-    return token
\ No newline at end of file
+    return token
+
+
+def get_apic_sdk_version(ip):
```

Comment:
how to use these?

for these and connector, please update doc as well.

Reply:
These functions are used internally by the connector, as follows:
For the connector to work properly, two libraries are needed:
1. acicobra
2. acimodel
Both libraries are accessible on any APIC's web-page, at https://<ip-or-hostname>/cobra/_downloads
The functions check if the required packages are installed and if their versions match the version of the APIC (the check is done during the init phase of the connector)
If any discrepancy is observed, the proper packages are obtained from the APIC and installed

Nonetheless, will update docs for these.

Reply:
sorry, seems my bad. I found the referenced points for these.